draft-patton-cfrg-vdaf.md

```diff
@@ -490,13 +494,18 @@ share or an error."}
   {{run-vdaf}}), and one of the input shares generated by the client
   (`input_share`). Its outputs is the Aggregator's initial preparation state.
 
-* `PrepState.next(inbound: Vec[Bytes]) -> outbound` is the deterministic
+* `PrepState.next(inbound: Optional[Bytes]) -> outbound` is the deterministic
   preparation-state update algorithm run by each Aggregator. It updates the
   Aggregator's preparation state (an instance of `PrepState`) and returns either
   its outbound message for the current round or, if this is the last round, its
   output share. An exception is raised if a valid output share could not be
-  recovered. The input of this algorithm is the sequence of inbound messages
-  from the previous round or, if this is the first round, an empty vector.
+  recovered. The input of this algorithm is the preprocessed inbound message or,
+  if this is the first round, `None`.
+
+* `prep_preprocess(agg_param, messages: Vec[Bytes]) -> preprocessed_message` is
+  a deterministic procedure used to preprocess the sequence of inbound messages
+  from the previous round into the input for the preparation-state update
+  algorithm.
```

Comment:
Can we think of a better verb than "preprocess"? While accurate, it seems too generic.

Reply:
I agree, in particular the double `pre` prefix is a bit confusing. Maybe `combine_verifier_shares` or something like that?

Reply:
I like the term `verifier_share`, but it's somewhat specific to schemes like `prio3` and `poplar1` where the output share are first recovered and then verified. At this stage I'd like to leave the door open for things like Prio+ that are based instead on oblivious transfer. (Output shares aren't known until the prep phase is complete.)

Reply:
I went with "prepare-message share", which is overly wordy but says what we want. The method is now called `prep_share_to_prep`, which is somewhat consistent with our naming conventions else where. 

Other suggestions for alternatives are welcome! (Naming things is hard.)